CIAOS-Rust/client/ciaos/client.py

```diff
@@ -0,0 +1,203 @@
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), 'util'))
+import requests
+from typing import List
+from util.flatbuffer.flatbuffer_handler import create_flatbuffer, parse_flatbuffer
+from config import API_URL
+
+class Ciaos:
+    def __init__(self, user: str):
+        """
+        Initialize Ciaos client with user.
+        API URL is imported from config.py
+
+        Args:
+            user (str): Username for API requests
+        """
+        self.user = user
+        self.headers = {
+                "User": self.user,
+            }
+
+    def put(self, key: str, data_list: List[bytes]):
```

Comment:
why is this still key, data_list I mentioned in the before comments that this should be 
put(key,path,data_list) either path or data_list is mandatory

Reply:
Orelse rename this to save and create two new function put which accepts only key.path

Reply:
hey so how will the put function with path work?
should i add path as another column in sql table 
if a key can have multiple paths
in get function should i provide path as optional so that when they dont pass a path , it retrieves all the files?

Reply:
Why do you need the path in the sql table?

Where have I added the comment? It's in the client file. Just like s3 user should be able to give a path and you take that file and convert it into flatbuffer and store it. If key is empty use the file name as key.

Reply:
hey updated the code, were these the expected changes ?